users/config.rst

```diff
@@ -25,11 +26,12 @@ Description
 
 Syncthing uses a single directory to store configuration and crypto keys.
 Syncthing also has a database, which is often stored in this directory too.
-The config location defaults to ``$HOME/.config/syncthing``
-(Unix-like), ``$HOME/Library/Application Support/Syncthing`` (Mac),
-or ``%LOCALAPPDATA%\Syncthing`` (Windows). It can be changed at runtime
-using the ``--config`` flag. In this directory the following files are
-located:
+The config location defaults to ``$XDG_STATE_HOME/syncthing`` or
```

Comment:
```suggestion
The location defaults to ``$XDG_STATE_HOME/syncthing`` or
```

I think talking especially about a "config location" here can be confusing after the previous statement.  Please reflow at will if accepting the suggestion. :wink: